packages/vue-app/template/middleware.js

```diff
@@ -1,8 +1,10 @@
 const middleware = {}
-<% middleware.forEach(m => {
-   const name = m.src.replace(new RegExp(`\\.(${extensions})$`), '')
+<% for (const m of middleware) {
+  const name = m.name || m.src.replace(new RegExp(`\\.(${extensions})$`), '')
```

Comment:
Maybe we can add a TODO comment to remove duplicate logic in next major version?

Reply:
done :)